README.md

```diff
@@ -2,109 +2,116 @@
  <img src="https://docs.freetubeapp.io/images/logoColor.png" width=500 align="center">
 </p>
 
-FreeTube is an open source desktop YouTube player built with privacy in mind.
-Use YouTube without advertisements and prevent Google from tracking you with their cookies and JavaScript.
-Available for Windows, Mac & Linux thanks to Electron.
+**Copylefted libre software desktop YouTube player** (with privacy).
 
-Please note that FreeTube is currently in Beta. While it should work well for
-most users, there are still bugs and missing features that need to be
-addressed.
+No advertisements or local Google claws tracking you with cookies and JavaScript.
 
-[Download FreeTube](https://github.com/FreeTubeApp/FreeTube/releases)
+Available for Linux, Windows and macOS thanks to Electron.
 
-### Browser Extension
+Should work for most users, but there are still bugs and missing features.
+
+[Download](https://github.com/FreeTubeApp/FreeTube/releases)
 
-FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect) extension, which will allow you to open YouTube links into FreeTube. You must enable the option within the advanced settings for it to work.
+### Browser Extension
 
-Download Privacy Redirect for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-redirect/) or [Google Chrome](https://chrome.google.com/webstore/detail/privacy-redirect/pmcmeagblkinmogikoikkdjiligflglb).
+FreeTube is supported by the [Privacy Redirect](https://github.com/SimonBrazell/privacy-redirect), on [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-redirect/) or [Google Chrome](https://chrome.google.com/webstore/detail/privacy-redirect/pmcmeagblkinmogikoikkdjiligflglb), \
+which allows you to open YouTube links into FreeTube from your web browser. \
+Turn it on from the advanced settings in the extention.
 
-If you have issues with the extension working with FreeTube, please create an issue in this repository instead of the extension repository.
+Only report issues getting FreeTube to work with the extention here.
 
 ## How does it work?
-FreeTube uses a built in extractor to grab and serve data / videos.  The [Invidious API](https://github.com/iv-org/invidious) can also optionally be used.  FreeTube does not use any official APIs to obtain data.  While YouTube can still see your video requests, it can no
-longer track you using cookies or JavaScript. Your subscriptions and history are stored locally on your computer and never sent out.  Using a VPN or Tor is highly recommended
-to hide your IP while using FreeTube.
+A built in extractor or the [Invidious API](https://github.com/iv-org/invidious) is used to grab and serve data/videos. \
```

Comment:
This phrasing is terser than the original but also less clear. I would advise starting the sentence with "FreeTube uses ..." and employing a more active tense.